src/assets/data/edition/series/1/section/5/m146/textcritics.json

```diff
@@ -1,3 +1,144 @@
 {
-    "textcritics": []
+    "textcritics": [
+        {
+            "id": "M_146_TF1",
+            "label": "M 146 einzige Textfassung",
+            "description": [
+                "Für die Edition der einzigen Textfassung von „Das lockere Saatgefilde“ M 146 ist <a (click)=\"ref.navigateToReportFragment({complexId: 'M_146', fragmentId: 'source_A'})\"><strong>A</strong></a> einzige relevante Quelle. <br /> Die in <a (click)=\"ref.navigateToReportFragment({complexId: 'M_146', fragmentId: 'source_A'})\"><strong>A</strong></a> unterscheidbaren Korrekturschichten wurden mit Tinte auf Rasur oder Tektur (<a (click)=\"ref.navigateToReportFragment({complexId: 'M_146', fragmentId: 'source_A_corr_1'})\">Korrekturen 1</a>) und mit Bleistift oder Buntstift (<a (click)=\"ref.navigateToReportFragment({complexId: 'M_146', fragmentId: 'source_A_corr_2'})\">Korrekturen 2</a>) vorgenommen. Der Zustand vor Korrekturen 1 ist nicht eindeutig zu entziffern. Der Zustand nach Korrekturen 1 etabliert eine kohärente Textfassung. Korrekturen 2 sind nicht immer eindeutig. <br /> Hauptquelle für die Textedition von „Das lockere Saatgefilde“ M 146 ist <a (click)=\"ref.navigateToReportFragment({complexId: 'M_146', fragmentId: 'source_A'})\"><strong>A</strong></a> inklusive Korrekturen 1."
+            ],
+            "comments": [
+                {
+                    "blockHeader": "",
+                    "blockComments": [
+                        {
+                            "svgGroupId": "TODO",
+                            "measure": "1",
+                            "system": "Ges.",
+                            "position": "2.–3. Note",
+                            "comment": "Text sic: Silbentrennung <em>lo-cke</em>(-re)."
+                        },
+                        {
+                            "svgGroupId": "TODO",
+                            "measure": "3",
+                            "system": "Ges.",
+                            "position": "2. Note",
+                            "comment": "Text: Großschreibung <em>Da</em> mit Blick auf <a (click)=\"ref.navigateToReportFragment({complexId: 'M146', fragmentId: 'text_George_DJdS'})\"><strong>George_DJdS</strong></a> (am Zeilenanfang). <a (click)=\"ref.navigateToReportFragment({complexId: 'M_146', fragmentId: 'source_A'})\"><strong>A</strong></a>: Kurrentschrift unspezifisch hinsichtlich Groß-/Kleinschreibung."
```

Comment:
Hier und im Folgenden George_DsR gemeint? Jedenfalls gibt es nur das in der source-list.